blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,9 @@ None
 Changes
 =======
 
+- Implemented the ``DROP ANALYZER`` statement to support removal of custom
+  analyzer definitions from the cluster.
```

Comment:
Should we also mention the `information_schema.routines.routine_definition` change?

Reply:
Wasn't sure if my solution is fine for everyone. If so, I'd make it visible and add a changes entry.